index.js

```diff
@@ -1,3 +1,11 @@
+const { formatBody } = require('./lib/formatBody');
+const { sendMail } = require('./lib/sendMail');
+
 exports.handler = async (event) => {
-  console.log(event);
+  const emailAddress = event.alerting.alerts[1].address;
```

Comment:
Do we know that the 'email' will always be at index 1? Maybe filter instead, might be less brittle?

Reply:
you should see the following in `index.js`:

```javascript
exports.handler = async (event) => {
  const { id: testId } = await getTestId(event.title);
  const alertChannels = await getAlertInfo(testId);

  const targets = [];

  alertChannels.forEach((target) => {
    switch (target.type) {
      case 'email':
        targets.push(new EmailClient(target.destination, event));
        break;
      case 'slack':
        console.log('slack implementaiton pending...');
        // targets.push(new SlackCLient(target));
        break;
      default:
        targets.push({ error: 'Unrecognized channel type' });
    }
  });

  targets.forEach((target) => target.send());
};

```